cognite/client/_api/simulators/models_revisions.py

```diff
@@ -21,6 +26,47 @@
     from cognite.client import ClientConfig, CogniteClient
 
 
+class SimulatorModelRevisionsDataAPI(APIClient):
```

Comment:
why new API class, if you can just it it into SimulatorModelRevisionsAPI?
it's only one endpoint